docs/deployment/allocation.md

```diff
@@ -67,19 +67,19 @@ Format[^1]: **`--time-limit <duration>`**
 Sets the walltime of created allocations.
 
 This parameter is **required**, as HyperQueue must know the duration of the individual allocations.
-Make sure that you pass a  time limit that does not exceed the limit of the PBS/Slurm queue
+Make sure that you pass a time limit that does not exceed the limit of the PBS/Slurm queue
 that you intend to use, otherwise the allocation submissions will fail. You can use the
 [`dry-run` command](#dry-run-command) to debug this.
 
 Workers in this allocation queue will be by default created with a time limit equal to the time limit of the queue
 (unless overridden with [Worker time limit](#worker-time-limit)).
 
 !!! Important
-    If you specify a [time request](../jobs/jobs.md#time-management)
-    for a task, you should be aware that the time limit for the allocation queue **should be larger than the time request**
-    if you want to run this task on workers created by this allocations queue, because it will always take some time before
-    a worker is fully initialized. For example, if you set `--time-request 1h` when submitting a task, and `--time-limit 1h`
-    when creating an allocation queue, this task will never get scheduled on workers from this queue.
+If you specify a [time request](../jobs/jobs.md#time-management)
+for a task, you should be aware that the time limit for the allocation queue **should be larger than the time request**
+if you want to run this task on workers created by this allocations queue, because it will always take some time before
+a worker is fully initialized. For example, if you set `--time-request 1h` when submitting a task, and `--time-limit 1h`
+when creating an allocation queue, this task will never get scheduled on workers from this queue.
```

Comment:
I think this block should be still indented, otherwise it will not render properly.

Reply:
Bloody IDE autoformatter.. thanks!